kernel/core/extend/userland.c

```diff
@@ -1,12 +1,21 @@
 #include <debug.h>
 
 #include <extend/userland.h>
+#include <extend/intr.h>
 
 void userland() {
    debug("This is fine.\n");
 
+   // Syscall
+   /*
+   ** TODO: change asm automatic
+   */
+   // asm volatile ("mov $1, %eax ; int $128\n\t");
+   uint32_t test = 42;
+   asm volatile ("mov %0, %%eax ; mov %1, %%ebx ; int %2\n\t" :: "i" (SYS_READ_CNT),"i" () , "i" (INT_SYSCALL) : "%eax", "%ebx");
```

Comment:
```suggestion
   int read_cnt = SYS_READ_CNT;
   int val = VALUE;
   asm volatile ("int %2\n\t" :: "a" (read_cnt), "b" (val) , "i" (INT_SYSCALL));
```
No need for clobbers here. You use clobbers when you use a register without declaring it in input or output